tests/test_push_docker.py

```diff
@@ -830,21 +850,28 @@ def test_push_docker_full_success(
 
     mock_get_docker_push_items.assert_called_once_with()
     mock_get_docker_push_items.assert_called_once_with()
-    mock_check_repos_validity.assert_called_once_with([container_multiarch_push_item])
-    mock_generate_backup_mapping.assert_called_once_with([container_multiarch_push_item])
+    mock_check_repos_validity.assert_called_once_with(
```

Comment:
Could you add a check if filter_unrealted_repos was called?

Reply:
it's not mocked so it has to be called